projects/kicks/index.js

```diff
@@ -0,0 +1,19 @@
+const STAKING = "0x7f435dC15CD9d686C48B6beD2cB8d32D895cfb93";
+const TOKEN = "0xfEB4e9B932eF708c498Cc997ABe51D0EE39300cf";
+
+async function staking(time, ethBlock, _b, { api }) {
+  const supply = await api.call({
+    abi: "function totalSupply() external view returns (uint256)",
```

Comment:
can you use erc20:balanceOf() instead?

Reply:
In fact the calculation with ERC20::balanceOf will be incorrect due to the fact that the whole reward amount stays on the contract too, until users claim it.  Which means that we have to separately count TVL by calling totalSupply (which counts only users` liquidity)